### PR TITLE
Merge Feature/create report into main

### PR DESCRIPTION
**Pull request by wafaa-rakchamber**
This is Wafaa's changes

### CODE REVIEW

File: report.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body{
+    background-color: gray;
```
Comment (mmorhaf) [inline report.css:2]: please add the color  as red.

File: report.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        
+    </head>
+    <body>
+            <h1> Reports from Wafaa </h1>
```
Comment (mmorhaf) [inline report.html]: make it H2 please

**Review by mmorhaf — CHANGES_REQUESTED** [review]

File: report.css (hunk 1 above)
Comment (wafaa-rakchamber) [reply report.css:2]: done

File: report.html (hunk 2 above)
Comment (wafaa-rakchamber) [reply report.html]: done

**Review by mmorhaf — APPROVED** [review]

**Pull request merged**